includes/LogisticsOrderFactory.php

```diff
@@ -80,8 +80,8 @@ public function buildFromWooCommerceOrder(object $wooCommerceOrder): CreateLogis
         $payload->setLogisticsOrder($logisticsOrder);
 
         $callbacks = new LogisticsOrderCallbacks();
-        $callbacks->setSuccessCallback($this->settingsRepository->getSiteUrl() . "/wp-json/gebrueder-weiss-woocommerce/v1/orders/" . $wooCommerceOrder->get_id() . "/callbacks/success");
-        $callbacks->setFulfillmentCallback($this->settingsRepository->getSiteUrl() . "/wp-json/gebrueder-weiss-woocommerce/v1/orders/" . $wooCommerceOrder->get_id() . "/callbacks/fulfillment");
+        $callbacks->setSuccessCallback($this->settingsRepository->getHomeUrl() . "/wp-json/gebrueder-weiss-woocommerce/v1/orders/" . $wooCommerceOrder->get_id() . "/callbacks/success");
```

Comment:
get home url is not correct here - what we would need is the rest url. otherwise it will never be correct, if someone uses rest_url filter. for example.

https://developer.wordpress.org/reference/functions/get_home_url/ 
vs
https://developer.wordpress.org/reference/functions/get_rest_url/